Lib/test/test_codeop.py

```diff
@@ -310,5 +311,11 @@ def test_warning(self):
             compile_command("0 is 0")
             self.assertEqual(len(w.warnings), 1)
 
+        # bpo-41520: check SyntaxWarning treated as an SyntaxError
+        with self.assertRaises(SyntaxError):
+            warnings.simplefilter('error', SyntaxWarning)
+            compile_command('1 is 1\n', symbol='exec')
```

Comment:
This source should not have '\n' as single lines are passed without it.  That is why to recompile with \n and \n\n added.

Reply:
Even w/o the `\n` 3.8.5 does not raise the SyntaxError.

```
$ cat foo.py
import warnings
from codeop import compile_command

warnings.simplefilter('error', SyntaxWarning)
res = compile_command('1 is 1', symbol='exec')
print('Res', res)

$ python foo.py
Res None
```

Reply:
I expect that, but aside from your issue, the test if better if it calls compile_command the way it is intended to be used and is used by code.InteractiveInterpreter.